db/migrate/20211124202133_add_average_score_.rb

```diff
@@ -0,0 +1,6 @@
+class AddAverageScore < ActiveRecord::Migration[6.1]
+  def change
+    add_column :departments, :average_score, :integer, default: 0
```

Comment:
We can discuss, but I would suggest making `average_score` a float, then passing along only two decimal points to the client.

Reply:
Good idea, updated to that